ops/charm.py

```diff
@@ -375,6 +384,7 @@ class CharmEvents(EventSetBase):
     install = EventSource(InstallEvent)
     start = EventSource(StartEvent)
     stop = EventSource(StopEvent)
+    remove = EventSource(RemoveEvent)
```

Comment:
while we're here, we could also do relation-created, though that is likely to be more involved, as it exposes an application name but not a unit (at this time, Gustavo and I have talked about trying to have a hint to the number of units.)

The actual list in 2.8 looks to be from:
https://github.com/juju/charm/blob/v6/hooks/hooks.go

Though IIRC LeaderDeposed was discussed but never implemented.

Also RelationBroken is being updated in 2.8 to add JUJU_REMOTE_APP, AIUI. 

But I think we have the rest covered.

Reply:
OK, I can add relation-created as well.

`leader-deposed` exists as a Juju "operation" but doesn't result in a hook being executed. I could add it in case it ever does but I would wait until an explicit discussion.

RelationBroken can be covered with a version check I think.